### PR TITLE
Token allocation, and migration scripts as dependencies

### PR DESCRIPTION
**Pull request by mikec**
This adds a migration script for allocating EtherToken, and changes some existing migration scripts so that they can be run from outside of the project

### CODE REVIEW

File: test/resources/add-token-pair/development/WETH_FCR.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  tokenB: {
+    symbol: 'WETH',
+    address: "0x4cadb4939da3368e89fbf6d42fd3e17811bd35b4",
+    // Check ETH oracle:
+    //   https://makerdao.com/feeds/#0x729d19f657bd0614b4985cf1d82531c67569197b
+    //   Price: 423.960
+    //   20$ = 20/423.960 ETH = 0.04717426172
+    funding: 10
+  },
+  // OMG
```
Comment (jpkcambridge) [inline test/resources/add-token-pair/development/WETH_FCR.js]: Can change this to 'FCR' and remove the commented out OMG code, I think

Comment (mikec) [reply test/resources/add-token-pair/development/WETH_FCR.js]: yep!

File: test/resources/add-token-pair/development/WETH_FCR.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+module.exports = {
+  // WETH
+  tokenB: {
+    symbol: 'WETH',
+    address: "0x4cadb4939da3368e89fbf6d42fd3e17811bd35b4",
+    // Check ETH oracle:
```
Comment (jpkcambridge) [inline test/resources/add-token-pair/development/WETH_FCR.js]: What is this?

File: src/truffle/add-token-pairs.js [hunk 3]
```diff
@@ -39,6 +39,7 @@ var argv = require('yargs')
   .argv
 
 async function addTokenPairs () {
+  console.log("here!!")
```
Comment (jpkcambridge) [inline src/truffle/add-token-pairs.js]: should this be cleaned up?

File: test/resources/add-token-pair/development/WETH_FCR.js (hunk 2 above)
Comment (mikec) [reply test/resources/add-token-pair/development/WETH_FCR.js]: something we don't need lol

File: src/truffle/add-token-pairs.js (hunk 3 above)
Comment (mikec) [reply src/truffle/add-token-pairs.js]: @jpkcambridge I removed it

**Comment by jpkcambridge** [comment]
👍 @mikec Looks good to me.

**Pull request merged**